application.go

```diff
@@ -109,6 +129,28 @@ type DeleteAppOpts struct {
 	Force bool `url:"force,omitempty"`
 }
 
+// Diego ======
+
+// SetIPAddressPerTask defines that the application will have a IP address defines by a external agent. this
+// configuration is not allowed to be used with Port or PortDefinitions.
+func (r *Application) SetIPAddressPerTask(ipAddressPerTask *IPAddressPerTask) *Application {
```

Comment:
I'd let the method take a non-pointer. It simplifies the usage while we shouldn't need pointer semantics for performance or other reasons. Plus, we'd be consistent with the other methods added by this PR.